open-banking-brasil-financial-api-1_ID1-ptbr.md

```diff
@@ -176,108 +176,107 @@ Para efeitos deste documento, os termos definidos em [RFC6749], [RFC6750], [RFC7
 
 #### Token de ID como assinatura separada  {#detached}
 
-O Servidor de Autorização deve suportar as disposições especificadas na cláusula 5.2.2.1 de [Financial-grade API Security Profile 1.0 - Parte 2: Advanced] [FAPI-1-Advanced]
+O Servidor de Autorização *deve* suportar as disposições especificadas na cláusula 5.2.2.1 de [Financial-grade API Security Profile 1.0 - Parte 2: Advanced] [FAPI-1-Advanced]
 
 Além disso, se o valor `response_type` `code id_token` for usado, o servidor de autorização:
 
-1. não deve retornar Informação de Identificação Pessoal (PII) confidenciais no token de ID na resposta de autorização, mas se for necessário,
-então ele deve criptografar o token de ID.
+1. **não deveria** retornar Informação de Identificação Pessoal (PII) confidenciais no token de ID na resposta de autorização, mas se for necessário, então ele **deve** criptografar o token de ID.
 
-#### Solicitando a reivindicação "cpf"  {#cpf}
+#### Solicitando uma "claim" **cpf**  {#cpf}
 
-Este perfil define "cpf" como uma nova reivindicação padrão de acordo com cláusula 5.1 [OIDC]
+Este perfil define "cpf" como uma nova `claim` padrão de acordo com cláusula 5.1 [OIDC]
 
 O número do **CPF** (Cadastro de Pessoas Físicas, [sepeˈɛfi]; português para "Registro de Pessoas Físicas") é o cadastro de pessoa física **brasileira**. Este número é atribuído pela Receita Federal **Brasileira** para brasileiros e estrangeiros residentes que, direta ou indiretamente, pagar impostos no **Brasil**.
 
 No modelo de identidade do Open Banking Brasil, o cpf é uma string composta por números 11 caracteres de comprimento e podem começar com 0.
-Se a reivindicação cpf for solicitada como uma reivindicação essencial para o token de ID ou resposta UserInfo com um parâmetros de valores solicitando um valor cpf específico, o Servidor de Autorização DEVE retornar um valor de reivindicação cpf que corresponde ao valor solicitado. Se esta for uma reivindicação essencial e o requisito não puder ser atendido,
-  então, o Authorization Server DEVE tratar esse resultado como uma tentativa de autenticação falhada.
+
+Se o parâmetro `"claims"` **cpf** for solicitado como essencial (`"essential":true`) para o ID token ou para a resposta ao endpoint de UserInfo e na solicitação constar no parâmetro `value` com determinado **CPF** exigido, o Authorization Server **deve** retornar no atributo **cpf** o valor que corresponda ao da solicitação. Se for uma "claim" indicada como essencial e não puder ser preenchido ou validado (CPF divergente do informado no `value`, por exemplo), o Authorization Server deve tratar a solicitação como uma tentativa de autenticação com falha.
+
+Se o parâmetro `"claims"` **cpf** for solicitado como essencial (`"essential=true"`) para constar no ID Token ou como resposta no endpoint de UserInfo, o Authorization Server deve retornar no atributo **cpf** o valor (_value_) com o **CPF** do usuário autenticado.
```

Comment:
essential 'true' vs essential: true. I would recommend against specifying how it is done - you really should leave it to the base specification for someone to read. i.e 'You say, if it is marked essential as per the specification.' you don't re-define how to mark it as essential as basic mistakes occur like this.

Reply:
I agree with you. It could cause more confusion than helps. I removed this examples/tips. Would you (or/and @alex-siqueira @DeiGratia33) check?